client/src/Pages/Home.vue

```diff
@@ -183,13 +200,20 @@
                     }
                 });
             },
+            bookingValue: function (value){
```

Comment:
I could not entirely understand this functionality, I think, I will consult you about it, tomorrow.